HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -1315,31 +1413,47 @@ def self.apply_infiltration_and_ventilation_fans(model, weather, vent_mech, vent
+    erv_hrv_cfm = vent_mech_erv_hrv.map { |vent_mech| vent_mech.flow_rate * (vent_mech.hours_in_operation / 24.0) }.inject(0.0, :+)
+    cfis_cfm = vent_mech_cfis.map { |vent_mech| vent_mech.flow_rate }.inject(0.0, :+)
+
+    # Combine exhaust and supply
+    combined_bal_cfm = [sup_cfm, exh_cfm].min
+    combined_unbal_cfm = sup_cfm - exh_cfm
+    # Balanced and unbalanced airflow calculated for hvac sizing
+    tot_bal_cfm = bal_cfm + erv_hrv_cfm + combined_bal_cfm
+    unbal_cfm = combined_unbal_cfm + cfis_cfm
 
-    equip_def = OpenStudio::Model::ElectricEquipmentDefinition.new(model)
-    equip_def.setName(Constants.ObjectNameMechanicalVentilationHouseFan)
-    equip = OpenStudio::Model::ElectricEquipment.new(equip_def)
-    equip.setName(Constants.ObjectNameMechanicalVentilationHouseFan)
-    equip.setSpace(@living_space)
-    equip_def.setFractionRadiant(0)
-    equip_def.setFractionLatent(0)
-    if vent_mech.nil?
-      equip_def.setFractionLost(1.0) # Arbitrary
-    elsif [HPXML::MechVentTypeExhaust].include? vent_mech.fan_type
-      equip_def.setFractionLost(1.0) # Fan heat does not enter space
-    elsif [HPXML::MechVentTypeSupply, HPXML::MechVentTypeCFIS].include? vent_mech.fan_type
-      equip_def.setFractionLost(0.0) # Fan heat does enter space
-    elsif [HPXML::MechVentTypeBalanced, HPXML::MechVentTypeERV, HPXML::MechVentTypeHRV].include? vent_mech.fan_type
-      equip_def.setFractionLost(0.5) # Supply fan heat enters space
-    end
-    equip.setSchedule(model.alwaysOnDiscreteSchedule)
-    equip.setEndUseSubcategory(Constants.ObjectNameMechanicalVentilation)
-    vent_mech_fan_actuator = OpenStudio::Model::EnergyManagementSystemActuator.new(equip, 'ElectricEquipment', 'Electric Power Level')
-    vent_mech_fan_actuator.setName("#{equip.name} act")
+    # Store info for HVAC Sizing measure
+    # Please review this, and its integration with hvac_sizing
+    model.getBuilding.additionalProperties.setFeature(Constants.SizingInfoMechVentWholeHouseRateBalanced, tot_bal_cfm)
+    model.getBuilding.additionalProperties.setFeature(Constants.SizingInfoMechVentWholeHouseRateUnbalanced, unbal_cfm.abs)
+    model.getBuilding.additionalProperties.setFeature(Constants.SizingInfoMechVentExist, (not vent_fans_mech.empty?))
+
+    # Fan Actuators
+    # Do we still want to create ee if there's no fan in HPXML? Reporting purpose?
+    sup_fan_actuator = add_ee_for_vent_fan_power(model, Constants.ObjectNameMechanicalVentilationHouseFanSupply, HPXML::MechVentTypeSupply)
+    exh_fan_actuator = add_ee_for_vent_fan_power(model, Constants.ObjectNameMechanicalVentilationHouseFanExhaust, HPXML::MechVentTypeExhaust)
+    bal_fan_actuator = add_ee_for_vent_fan_power(model, Constants.ObjectNameMechanicalVentilationHouseFanBalanced, HPXML::MechVentTypeBalanced)
+    cfis_fan_actuator = add_ee_for_vent_fan_power(model, Constants.ObjectNameMechanicalVentilationHouseFanCFIS, HPXML::MechVentTypeCFIS)
```

Comment:
I suggest simplifying this. At the end of the day, we need: 
1) one EE object for CFIS systems (since it has FractionLost=0.0 and needs to be actuated), and
2) one EE object for all non-CFIS systems (where we don't need to use EMS/actuator and we can calculate FractionLost from the various mech vent system types and their fan powers)